src/timer/TimerProvider.jsx

```diff
@@ -52,15 +49,19 @@ const TimerProvider = ({
     },
   ).join(':');
 
-  const pollExam = () => {
+  const pollExam = useCallback(() => {
```

Comment:
This function was created each render, which happens each second. the `useCallback()` is to prevent this for efficiency.